### PR TITLE
Fix: Change metric from driver to platform

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

File: core/payment/src/processor.rs [hunk 1]
```diff
@@ -388,7 +388,7 @@ impl PaymentProcessor {
             activity_payment.allocation_id = None;
         }
 
-        counter!("payment.amount.sent", ya_metrics::utils::cryptocurrency_to_u64(&msg.amount), "driver" => msg.driver);
+        counter!("payment.amount.sent", ya_metrics::utils::cryptocurrency_to_u64(&msg.amount), "platform" => msg.driver);
```
Comment (Wiezzel) [inline core/payment/src/processor.rs:391]: The value should be `payment_platform` not `msg.driver` then.

**Review by Wiezzel — CHANGES_REQUESTED** [review]

File: core/payment/src/processor.rs (hunk 1 above)
Comment (nieznanysprawiciel) [reply core/payment/src/processor.rs]: Thanks for noticing

Comment (Wiezzel) [reply core/payment/src/processor.rs]: Now you're using a moved value. Compile before committing. :P

Comment (nieznanysprawiciel) [reply core/payment/src/processor.rs]: Eh. I hoped it will be easy fix :P

**Review by Wiezzel — APPROVED** [review]

**Pull request merged**